### PR TITLE
Appadmin html formatting, adding bootstrap 3 classes, other minor visual improvements

### PR DESCRIPTION
**Pull request by amerikan**
Updates the Welcome app

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,4 @@ HOWTO-web2py-devel
 *.sublime-project
 *.sublime-workspace
 .idea/*
+*.DS_Store
```
Comment (amerikan) [inline .gitignore]: OSX generates these

File: applications/welcome/controllers/appadmin.py [hunk 2]
```diff
@@ -227,17 +227,26 @@ def select():
...
-                requires=IS_NOT_EMPTY(
-                    error_message=T("Cannot be empty")))), TR(T('Update:'),
-                INPUT(_name='update_check', _type='checkbox',
-                value=False), INPUT(_style='width:400px',
-                _name='update_fields', _value=request.vars.update_fields
-                                    or '')), TR(T('Delete:'), INPUT(_name='delete_check',
-                _class='delete', _type='checkbox', value=False), ''),
-                TR('', '', INPUT(_type='submit', _value=T('submit')))),
-                _action=URL(r=request, args=request.args))
+    form = FORM(
```
Comment (amerikan) [inline applications/welcome/controllers/appadmin.py]: Used Boostrap `form` formatting rather than `html` tables.

**Comment by amerikan** [comment]
There are about 3 parts where I changed the text in `T('something')` not sure if I would have to go through every single language file and make the change?

Also, for some reason even after changing the style, when I go to the admin interface I create a new app it doesn't get the **welcome**'s app style. Is there somewhere I must do this manually so that it can start copying it correctly?

File: applications/welcome/views/appadmin.html [hunk 3]
```diff
@@ -1,278 +1,384 @@
 {{extend 'layout.html'}}
-<script><!--
-    jQuery(document).ready(function(){
-    jQuery("table.sortable tbody tr").mouseover( function() {
-    jQuery(this).addClass("highlight"); }).mouseout( function() {
-    jQuery(this).removeClass("highlight"); });
-    jQuery('table.sortable tbody tr:odd').addClass('odd');
-    jQuery('table.sortable tbody tr:even').addClass('even');
-});
-//--></script>
```
Comment (amerikan) [inline applications/welcome/views/appadmin.html]: Boostrap striped and hover classes do this job so I used them instead.

File: .gitignore (hunk 1 above)
Comment (mpranjic) [inline .gitignore]: you're missing a new line at the end of file (which you can see if you roll the mouse over)

**Comment by amerikan** [comment]
@niphlod @mdipierro anything I can do to get this reviewed and merge? Other please close.

**Pull request closed without merging**